helm-charts/README.md

```diff
@@ -1,126 +1,389 @@
-# Digger Backend Helm Chart
+# OpenTaco Helm Charts
 
-This Helm chart deploys the Digger backend orchestrator for managing Terraform/OpenTofu runs in your CI/CD pipelines.
+Production-ready Kubernetes deployment for the OpenTaco infrastructure management platform.
 
-## Installation
-
-### Install from GitHub Container Registry
+## Quick Start
 
 ```bash
-# Install directly
-helm install digger-backend oci://ghcr.io/diggerhq/helm-charts/digger-backend \
-  --namespace digger \
-  --create-namespace \
-  --values values.yaml
+# 1. Configure values file (see Configuration Checklist below)
+cp opentaco/values-test.yaml.example opentaco/values-test.yaml
```

Comment:
I think we should not expect people to clone the repo and use local files, we can instead just ask them to fetch the values from the published repo on ghcr.io like the step here https://www.runatlantis.io/docs/deployment.html#kubernetes-helm-chart

Reply:
yeah that makes sense